test/suite/test_app_thread_evict01.py

```diff
@@ -52,47 +51,31 @@ def get_stat(self, stat):
         stat_cursor.close()
         return val
 
-    def insert_range(self,uri,rows):
-        cursor = self.session.open_cursor(uri)
-
-        for i in range(rows):
-            self.session.begin_transaction()
-            cursor.set_key(i+1)
-            value = simple_value(cursor, i) + 'abcdef' * random.randint(500,1000)
-            cursor.set_value((value))
-            cursor.insert()
-            self.session.commit_transaction()
-
     def test_app_thread_evict01(self):
-        self.skipTest("This test fails randomly when it cannot pull application threads to perform eviction.")
-        num_app_evict_snapshot_refreshed = 0
-
         format='key_format={},value_format={}'.format(self.key_format, self.value_format)
         self.session.create(self.uri, format)
-        # Create our table.
-        ds = SimpleDataSet(self, self.uri, 1000, key_format=self.key_format, value_format=self.value_format)
-        ds.populate()
-
-        # Start a long running transaction.
-        session2 = self.conn.open_session()
-        session2.begin_transaction()
-        cursor = self.session.open_cursor(self.uri)
-        key = 350
-        cursor.set_key(key)
-        cursor.set_value(str(key))
-        cursor.insert()
 
         # For our target stat to be incremented we need our application thread to evict a page, but
         # this is probabilistic as the application thread is always racing against the internal
         # eviction threads. Give the application thread a few chances to beat the internal thread
-        for _ in range(0, 10):
-            for i in range(500):
-                self.insert_range(self.uri, i)
+        for _ in range(0, 20):
```

Comment:
For my interest, not a PR blocker. How often do we fail to evict pages on the first iteration of the loop?